Makefile

```diff
@@ -37,30 +36,14 @@ reload-rules:
 podman:
 	podman run --rm --mount type=bind,source=$(CURDIR),target=/src --mount type=bind,source=$(CURDIR)/../tkey-libs,target=/tkey-libs -w /src -it ghcr.io/tillitis/tkey-builder:2 make -j
 
-.PHONY: apps
-apps:
-	$(MAKE) -C apps
-
-# .PHONY to let go-build handle deps and rebuilds
-.PHONY: tkey-sign
-tkey-sign:
-	go build -ldflags "-X main.signerAppNoTouch=$(TKEY_SIGNER_APP_NO_TOUCH)" ./cmd/tkey-sign
-
-runsign.sh: apps/signer/runsign.sh
-	cp -af $< $@
-
-.PHONY: runtimer
-runtimer:
-	go build ./cmd/runtimer
```

Comment:
`cmd/runtimer/main.go` seems to still exist in the repo.

Reply:
Thanks. Removed.